File: clean_test_tweets.r
test_data <- read.csv('short_train.csv', header=TRUE)

#delete fold column
test_data[4] <- NULL

#delete region column
test_data[3] <- NULL

#rename columns
names(test_data) <- c("User", "Tweet")

write.csv(test_data, file="test_clean.csv")

File: clean_test_tweets.py
short_train <- read.csv('short_train.csv', header=TRUE)

#delete unnecessary columns
short_train[1] <- NULL

#remove ngrams containing @user_
regexp <- "@[a-zA-Z0-9_]*"
gsubtry <- gsub(pattern = regexp, replacement = "", x = short_train$Tweet)

#merge gsubtry back into short_train, rename as Tweet
short_train_clean <- cbind(short_train, gsubtry)
short_train_clean[2] <- NULL
names(short_train_clean)[3] <- "Tweet"